package.json

```diff
@@ -17,9 +17,9 @@
   "scripts": {
     "coverage": "istanbul cover node_modules/mocha/bin/_mocha -r test/fixtures/env.js test/**/*.js",
     "start": "node index",
-    "build": "npm run build:font-awesome && npm run build:grunt && npm run build:handlebars",
+    "build": "npm-run-all build:*",
     "build:font-awesome": "node scripts/build-fontawesome.js",
-    "build:grunt": "grunt",
+    "build:libs": "uglifyjs client/js/libs/*.js client/js/libs/jquery/*.js client/js/libs/handlebars/*.js -o client/js/libs.min.js --source-map client/js/libs.min.js.map --source-map-url libs.min.js.map -p relative",
```

Comment:
Alright, stupid question: do we *want* to uglify these? I don't think it matters too much in terms of performance.
EDIT: I also realize it makes things much simpler to load from the HTML file, so feel free to ignore if stupid comment :)